polkadot-protocol/protocol-components/transactions/transactions-formats.md

```diff
@@ -0,0 +1,104 @@
+
+`<signing account ID> + <signature> + <additional data>`
+
+When submitting a signed transaction, the signature is constructed by signing:
+
+- The actual call is composed of the following:
+    - The index of the pallet
+    - The index of the function call in the pallet
+    - The parameters required by the function call are being targeted
+
+- Some extra information, verified by the signed extensions of the transaction:
+    - What's the era for this transaction, i.e., how long should this call last in the transaction pool before it gets discarded?
+    - The nonce, i.e., how many prior transactions have occurred from this account? This helps protect against replay attacks or accidental double-submissions
+    - The tip amount paid to the block producer to help incentivize it to include this transaction in the block
+
+Then, some additional data that's not part of what gets signed is required, which includes:
+
+- The spec version and the transaction version ensure that the transaction is submitted to a compatible runtime
+- The genesis hash, which ensures that the transaction is valid for the correct chain
+- The block hash, that corresponds to the hash of the checkpoint block, enables the signature to verify that the transaction doesn't execute on the wrong fork by checking against the block number provided by the era information
+
+This process can be broken down into the following steps:
+
+1. Construct the unsigned payload
+2. Create a signing payload
+3. Sign the payload
+4. Serialize the signed payload
+5. Submit the serialized transaction
+
+An extrinsic is encoded into the following sequence of bytes just before being hex-encoded:
+
+`[ 1 ] + [ 2 ] + [ 3 ] + [ 4 ]`
+
+Where:
+
+- `[1]` contains the compact encoded length in bytes of all of the following data. Learn how compact encoding works using SCALE
+- `[2]` is a u8 containing 1 byte to indicate whether the transaction is signed or unsigned (1 bit) and the encoded transaction version ID (7 bits)
+- `[3]` if a signature is present, this field contains an account ID, an SR25519 signature, and some extra data. If unsigned, this field contains 0 bytes
+- `[4]` is the encoded call data. This comprises 1 byte denoting the pallet to call into, 1 byte denoting the call to make in that pallet, and as many bytes as needed to encode the arguments expected by that call
+
+The metadata interface provides the way applications know how to construct a transaction correctly. An application will learn how to correctly encode a transaction using metadata types and transaction format. If a call doesn't need to be signed, then the first bit in `[2]` will be 0, so an application will know not to try decoding a signature.
+
+## Signed Extensions
```

Comment:
There is also a reference doc in the Polkadot SDK docs that would be good to link here, and also contains most important types: https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/signed_extensions/index.html

Reply:
Thanks!! <3